packages/cli/src/aws/utils.ts

```diff
@@ -69,7 +70,11 @@ export async function getStackDetails(stackName: string): Promise<MedplumStackDe
   const result = {} as Partial<MedplumStackDetails>;
   await buildStackDetails(cloudFormationClient, stackName, result);
   if ((await cloudFormationClient.config.region()) !== 'us-east-1') {
-    await buildStackDetails(new CloudFormationClient({ region: 'us-east-1' }), stackName + '-us-east-1', result);
+    try {
+      await buildStackDetails(new CloudFormationClient({ region: 'us-east-1' }), stackName + '-us-east-1', result);
+    } catch {
+      // Fail gracefully
```

Comment:
What is the case where this throws?  Can we print a warning?

The only reason I'm asking is that "printing more details" has be a theme during recent self-hosting customer support sessions.  See: https://github.com/medplum/medplum/issues/3117